crypt.go

```diff
@@ -795,10 +862,11 @@ func addKeyToLUKS2Container(devicePath string, existingKey, key []byte, extraOpt
 // key argument.
 //
 // The recovery key is provided via the recoveryKey argument and must be a cryptographically secure 16-byte number.
-func AddRecoveryKeyToLUKS2Container(devicePath string, key []byte, recoveryKey RecoveryKey) error {
-	return addKeyToLUKS2Container(devicePath, key, recoveryKey[:], []string{
-		// use argon2i as the KDF with an increased cost
-		"--pbkdf", "argon2i", "--iter-time", "5000"})
+func AddRecoveryKeyToLUKS2Container(devicePath string, key []byte, recoveryKey RecoveryKey, options *KDFOptions) error {
+	if options == nil {
+		options = &KDFOptions{}
+	}
+	return addKeyToLUKS2Container(devicePath, key, recoveryKey[:], options, nil)
```

Comment:
In master, one cannot pass in an options struct, but instead recoverykey is hardcoded to be 5*seconds target duration for any slot.